src/test/kotlin/uk/gov/dluhc/printapi/logging/CorrelationIdMdcIntegrationTest.kt

```diff
@@ -132,43 +136,18 @@ internal class CorrelationIdMdcIntegrationTest : IntegrationTest() {
 
             // Then
             await.atMost(10, TimeUnit.SECONDS).untilAsserted {
-                val logEvent = TestLogAppender.getLogEventMatchingRegex(
-                    "Looking for certificate Print Requests to assign to a new batch",
-                    Level.INFO
-                )
-                assertThat(logEvent).isNotNull
-                val expectedCorrelationId = logEvent!!.mdcPropertyMap[CORRELATION_ID]
-
-                assertThat(
-                    TestLogAppender.getLogEventMatchingRegex(
-                        "Certificate ids \\[$certificateId\\] assigned to batch \\[.{32}\\]",
-                        Level.INFO
-                    )
-                ).hasCorrelationId(expectedCorrelationId)
-                assertThat(
-                    TestLogAppender.getLogEventMatchingRegex(
-                        "Batch \\[.{32}\\] containing .{1} print requests submitted to queue",
-                        Level.INFO
-                    )
-                ).hasCorrelationId(expectedCorrelationId)
-                assertThat(
-                    TestLogAppender.getLogEventMatchingRegex(
-                        "Completed batching certificate Print Requests",
-                        Level.INFO
-                    )
-                ).hasCorrelationId(expectedCorrelationId)
```

Comment:
As there is a race condition and frequently the flow aborts with an `InsufficientPrintRequestsInBatchException` error. This test fails, but the intention of the test is that the correlation id is the same in the logs, so propose getting a minimum of 4 logs and verifying that they have the same correlation ID.